rickled/__init__.py

```diff
@@ -262,7 +281,11 @@ def __init__(self, base : Union[dict,str,TextIOWrapper,list] = None, deep : bool
             for file in base:
                 with open(file, 'r') as f:
                     stringed = f'{stringed}\n{f.read()}'
+        elif 'path' in init_args and os.path.isfile(os.path.join(init_args['path'], base)):
```

Comment:
@FabianSperrle I'm not sure what the use case is here? Could you explain why this is useful?

Reply:
I am using the library in a FastAPI application and had issues with file paths being relative to the wrong files. In any case, as you have spotted, I continued with some quick and dirty development in my fork. I will set up a cleaned branch to merge into `dev` here in the next days.